packages/suite-base/src/panels/DiagnosticSummary/DiagnosticNodeRow.test.tsx

```diff
@@ -0,0 +1,49 @@
+/** @jest-environment jsdom */
+
+// SPDX-FileCopyrightText: Copyright (C) 2023-2024 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)<lichtblick@bmwgroup.com>
+// SPDX-License-Identifier: MPL-2.0
+
+import "@testing-library/jest-dom";
+import { fireEvent, render, screen } from "@testing-library/react";
+
+import DiagnosticNodeRow from "@lichtblick/suite-base/panels/DiagnosticSummary/DiagnosticNodeRow";
+import { NodeRowProps } from "@lichtblick/suite-base/panels/DiagnosticSummary/types";
+import BasicBuilder from "@lichtblick/suite-base/testing/builders/BasicBuilder";
+import DiagnosticsBuilder from "@lichtblick/suite-base/testing/builders/DiagnosticsBuilder";
+
+describe("DiagnosticNodeRow", () => {
+  let info: ReturnType<typeof DiagnosticsBuilder.info>;
+  let isPinned: boolean;
+  let onClick: jest.Mock;
+  let onClickPin: jest.Mock;
```

Comment:
remove from beforeEach() and transform to const
```suggestion
  const onClick: jest.Mock = jest.fn();
  const onClickPin: jest.Mock = jest.fn();
```

Reply:
Implemented